.env-webserver-garbage-collector

```diff
@@ -0,0 +1,37 @@
+#
+# Explicit plugins DISABLED in the webserver to create a garbage-collector
+# Docs plugins config of services/web/server/src/simcore_service_webserver/application_settings.py
+#
+
+
+WEBSERVER_ACTIVITY=null
+WEBSERVER_CATALOG=null
+WEBSERVER_COMPUTATION=null
+WEBSERVER_DIAGNOSTICS=null
+#WEBSERVER_DIRECTOR_V2 from .env

```

Comment:
what does this mean?

Reply:
Here I mapped all the plugin variables and explicitly set to ``null`` those that need to be disconnected. For those enabled, i just commented them. So to some extent, this env-file is **only to disable** plugins